### PR TITLE
HTML & CSS practice: Popup

### PR DESCRIPTION
**Pull request by Shizopo**
### HTML & CSS practice: Popup

[Demo](https://shizopo.github.io/html-css-popup/index.html) |
[Code base](https://github.com/Shizopo/html-css-popup)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Shizopo/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="./styles/index.css">
+    <title>Wooble</title>
+</head>
+<body>
+    <div class="header">
+        <a href="#" class="header-link" title="Your light and secure mail">Wmail</a>
+        <a href="#" class="header-link" title="Your photo gallery">Photos</a>
```
Comment (madmaxWMFU) [inline submissions/Shizopo/html-css-popup/index.html]: Use the special `<header>` tag to display the menu

File: submissions/Shizopo/html-css-popup/styles/index.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+}
+
+.more-apps:before {
+    content: "Show more";
+}
+
+.more-apps-checkbox:checked + .more-apps:before {
+    content: "Hide";
+    padding-left: .5em;
+}
```
Comment (madmaxWMFU) [inline submissions/Shizopo/html-css-popup/styles/index.css]: Hide button after opening additional programs.

File: submissions/Shizopo/html-css-popup/index.html (hunk 1 above)
Comment (Shizopo) [reply submissions/Shizopo/html-css-popup/index.html]: Done

File: submissions/Shizopo/html-css-popup/styles/index.css (hunk 2 above)
Comment (Shizopo) [reply submissions/Shizopo/html-css-popup/styles/index.css]: Also done!

**Comment by madmaxWMFU** [comment]
Nice job!

**Comment by A-Ostrovnyy** [comment]
Well done!

**Comment by Shizopo** [comment]
@madmaxWMFU @A-Ostrovnyy thank you for your reviews!
@o-msh is this PR ready to merge or there is something that I missed?

File: submissions/Shizopo/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        <input type="checkbox" id="apps-checkbox" class="apps-menu-checkbox visually-hidden">
+        <label for="apps-checkbox" class="apps-menu-button" title="Your Wooble apps">
+            <img src="./img/menu-icon.svg" class="apps-menu-icon">
+        </label>
+        <div class="apps">
+            <ul class="apps-list">
+                <li>
+                    <a href="#" class="app-link" title="Mail">
+                        <div class="app-link-logo">M</div>
```
Comment (A-Ostrovnyy) [inline submissions/Shizopo/html-css-popup/index.html]: Why you use `div` in this place and into other similar places you use `span`?

Comment (Shizopo) [reply submissions/Shizopo/html-css-popup/index.html]: My bad! Unfortunately missed it. Now it's fixed, see commits below.

**Comment by A-Ostrovnyy** [comment]
Good work!

**Pull request merged**